contributing.md

```diff
@@ -265,13 +265,17 @@ Below are the steps to add a new link:
 
 When you add an example to the [examples](examples) directory, please follow these guidelines to ensure high quality examples:
 
-- TypeScript should be leveraged for new examples (no need for separate JavaScript and TypeScript examples)
+- TypeScript should be leveraged for new examples (no need for separate JavaScript and TypeScript examples, converting old JavaScript examples is preferred)
```

Comment:
Do we want to keep few very simple example as pure js, such as hello world, that users can start quickly without any typescript related configuration.